import_flow_events.py

```diff
@@ -54,22 +59,63 @@
       utmTerm VARCHAR(40)
     );
 """
+Q_CREATE_METADATA_TABLE = """
+    CREATE TABLE IF NOT EXISTS flow_metadata (
+      flowId VARCHAR(64) NOT NULL UNIQUE SORTKEY,
+      beginTime TIMESTAMP NOT NULL,
+      duration INTERVAL NOT NULL,
+      completed BOOLEAN NOT NULL,
+      newAccount BOOLEAN NOT NULL,
+      browser VARCHAR(40),
+      browserVersion VARCHAR(40),
+      os VARCHAR(40),
```

Comment:
OK; we could also consider `ua` as a shorter prefix, but I'm going to resist the temptation to bikeshed - pick whichever feels right to you.

We should consider updating the activity-events tables for consistency, and may need to adjust any existing re:dash graphs that use this columns.